### PR TITLE
Fix #23, fix #31, fix #33. Add bar chart, calendar, cycle buttons and style front page

### LINKED ISSUE #23: Create a bar chart component

### LINKED ISSUE #31: Create a calendar component

### LINKED ISSUE #33: Cycle between charts

### PR DESCRIPTION
**Pull request by Danar435**

### CODE REVIEW

**Comment by Danar435** [comment]
I decided to stay up for another hour and add some logic behind the buttons, now you should be able to cycle between the different charts we have

File: components/client/bar-chart.tsx [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+      indexBy="day"
+      margin={{ top: 60, right: 90, bottom: 90, left: 90 }}
+      padding={0.3}
+      innerPadding={2}
+      borderRadius={6}
+      enableGridX={true}
+      enableGridY={false}
+      valueScale={{ type: "linear" }}
+      indexScale={{ type: "band", round: true }}
+      /* eslint-disable-next-line @typescript-eslint/prefer-readonly-parameter-types */
```
Comment (audio-engineer) [inline components/client/bar-chart.tsx:88]: @Danar435 Line comments in JavaScript/TypeScript are usually `//`. Block comments are `/** ... */`. `/* ... */` works but is unconventional.

File: components/client/home.tsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
+"use client";
```
Comment (audio-engineer) [inline components/client/home.tsx:1]: @Danar435 See comment above.

File: components/client/home.tsx [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+import IconButton from "@mui/material/IconButton";
+import ArrowBackIosNewIcon from "@mui/icons-material/ArrowBackIosNew";
+import ArrowForwardIosIcon from "@mui/icons-material/ArrowForwardIos";
+
+const Home = (): ReactElement => {
+  const components = [
+    <BarChart key="first" />,
+    <Heatmap key="second" />,
+    <OrderGrid key="third" />,
+  ];
```
Comment (audio-engineer) [inline components/client/home.tsx:19]: @Danar435 bad, bad... Markup belongs to markup (`return()`)

File: components/client/home.tsx [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+  const InitialPointer = 0;
+  const MinPointer = 0;
+  const PointerIncrement = 1;
```
Comment (audio-engineer) [inline components/client/home.tsx:23]: @Danar435 is this a type or a variable? Variables use camelCase, types PascalCase

File: components/client/home.tsx [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+
+  const [pointer, setPointer] = useState(InitialPointer);
```
Comment (audio-engineer) [inline components/client/home.tsx:25]: @Danar435 Ok now I see the only purpose of `initialPointer` is to set the default state. Just use a `0` literal. Rule of thumb: Only if you use a value more than once should you need a variable or constant.

**Review by audio-engineer — CHANGES_REQUESTED** [review]
A couple of changes requested. Do the corrections and we will merge it. You can remove the arrow buttons and do the other changes that we talked about yesterday in a future commit.

**Comment by audio-engineer** [comment]
@Danar435 I suggest you close this PR. I already implemented all of your changes myself, making this commit redundant.

**Pull request closed without merging**